twill-api/src/main/java/org/apache/twill/api/TwillPreparer.java

```diff
@@ -227,7 +227,7 @@
   TwillPreparer addSecureStore(SecureStore secureStore);
 
   /**
-   * Set the log level for Twill applications running in a container.
+   * Set the root log level for Twill applications in all containers.
```

Comment:
Same as above, do we have a way to set the root log level for a particular runnablE?